server/src/main/java/com/defold/extender/ProcessExecutor.java

```diff
@@ -127,7 +133,10 @@ public Void call() throws Exception {
             } else if (e.getCause() instanceof InterruptedException) {
                 throw (InterruptedException)e.getCause();
             } else {
-                throw new ExtenderException(e.getCause().toString());
+                StringWriter sw = new StringWriter();
+                PrintWriter pw = new PrintWriter(sw);
+                e.printStackTrace(pw);
+                throw new ExtenderException(sw.toString());
```

Comment:
This actually prints out the full callstack of the error.
I guess the cause, had another cause and we didn't iterate it?